exporter/jaeger/example_test.go

```diff
@@ -33,3 +33,16 @@ func Example() {
 	}
 	trace.RegisterExporter(exporter)
 }
+
+func ExampleNewExporter_agent() {
+	// Register the Jaeger exporter to be able to retrieve
+	// the collected spans.
+	exporter, err := jaeger.NewExporter(jaeger.Options{
+		AgentEndpoint: "localhost:6831",
```

Comment:
Agent can listen on TCP, right? Are we doing anything to allow user to depend on an TCP endpoint.

Reply:
@rakyll, As far as I know, agent only works on UDP. http://jaeger.readthedocs.io/en/latest/deployment/